tests/test_places.py

```diff
@@ -165,11 +166,12 @@ def test_place_detail(self):
             "ChIJN1t_tDeuEmsRUsoyG83frY4",
             fields=["business_status", "geometry/location", "place_id"],
             language=self.language,
+            reviews_sort="newest",
```

Comment:
I think we can set this to `self.reviews_sort` from your L39

Reply:
This has been updated, thank you.